sensor/kubernetes/listener/resources/complianceoperator/dispatchers/complianceoperatorprofiles.go

```diff
@@ -38,8 +39,50 @@ func (c *ProfileDispatcher) ProcessEvent(obj, _ interface{}, action central.Reso
 		return nil
 	}
 
+	// For nextgen compliance this ID is used to tell us which clusters have which profiles.  It is also
+	// useful for the deduping from sensor.
+	uid := string(complianceProfile.UID)
+
+	// We probably could have gotten away with re-using the storage proto here for the time being.
+	// But we have a new field coming on for profiles and using the storage object even in an internal api
+	// is a bad practice, so we will make that split now.  V1 and V2 compliance will both need to work for a period
+	// of time.  However, we should not need to send the same profile twice, the pipeline can convert the V2 sensor message
+	// so V1 and V2 objects can both be stored.
+	if features.ComplianceEnhancements.Enabled() {
```

Comment:
I wonder if feature flags are enough to guard this feature. Maybe you want to think about adding a capability from Central as well?

If the Central version is older than Sensor's, and you have this FF enabled, sensor will send `ComplianceOperatorProfileV2` but Central is likely expecting `ComplianceOperatorProfile`. 

Feature Flags in Sensor have this downside that they most of the times rely on Central having the feature enabled as well. And in this cases, a Central capability works better.

Reply:
@fredrb Does that sound like a reasonable plan, especially since compatibility test passed?

Reply:
Sure. These compatibility issues can be nasty if released. So as long as this ticket is addressed before releasing this feature it should be ok.

Reply:
Sounds good.  I have a goal of this sprint for that, while it is fresh in my mind.  But certainly before we release the feature